frontend/src/pages/TrainingReports/components/EventCard.js

```diff
@@ -55,6 +58,23 @@ function EventCard({
   || data.status === TRAINING_REPORT_STATUSES.SUSPENDED);
   const menuItems = [];
 
+  const canCompleteEvent = (() => {
+    if (!isOwner && !isNotCompleteOrSuspended) {
```

Comment:
The ! notCompletedOrSuspended is also sort of confusing here (double not).

Reply:
See above